src/coreclr/src/tools/Common/JitInterface/ThunkGenerator/ThunkGenerator.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFramework>$(NetCoreAppCurrent)</TargetFramework>
+    <TargetFramework>net5.0</TargetFramework>
```

Comment:
Why are all these tools staying on 5.0?  Is it because they need to run on the current tool framework and that's still 5.0?

Reply:
Is it because VS doesn't support net6.0 yet?

Reply:
Would it be better to define a `$(NetCoreToolCurrent)` or similar, so its a central replace for tools vs libraries, both now and in the future?